src/commonTest/kotlin/kotools/types/number/StrictlyNegativeIntTest.kt

```diff
@@ -71,6 +72,39 @@ class StrictlyNegativeIntTest {
             .shouldHaveAMessage()
     }
 
+    @ExperimentalNumberApi
+    @Test
+    fun number_toStrictlyNegativeIntOrNull_should_not_be_null_with_a_strictly_negative_Int() {
+        val number: Number = StrictlyNegativeInt.random().toInt()
+        val result: StrictlyNegativeInt? = number.toStrictlyNegativeIntOrNull()
+        result?.toInt() shouldEqual number
+    }
+
+    @ExperimentalNumberApi
+    @Test
+    fun number_toStrictlyNegativeIntOrNull_should_be_null_with_a_positive_Int() {
+        val number: Number = PositiveInt.random().toInt()
+        val result: StrictlyNegativeInt? = number.toStrictlyNegativeIntOrNull()
+        assertNull(result)
+    }
+
+    @ExperimentalNumberApi
+    @Test
+    fun number_toStrictlyNegativeIntOrThrow_should_pass_with_a_negative_Int() {
+        val number: Number = StrictlyNegativeInt.random().toInt()
+        number.toStrictlyNegativeIntOrThrow().toInt() shouldEqual number
```

Comment:
Declaring a `result` variable is useful for checking the return type of the tested function.

```suggestion
        val result: StrictlyNegativeInt = number.toStrictlyNegativeIntOrThrow()
        result.toInt() shouldEqual number
```